backend/dataall/base/api/gql/graphql_argument.py

```diff
@@ -8,23 +8,29 @@
 
 
 class Argument:
-    def __init__(self, name, type):
+    def __init__(self, name, type, description=''):
         self.name = name
+        self.description = description
         if isinstance(get_named_type(type), (Scalar, InputType, Ref, Enum)):
             self.type = type  # get_named_type(type)
         else:
             raise Exception('Invalid Argument Type')
 
     def gql(self):
+        description_str = ''
+        n = '\n'
```

Comment:
Do we really need '\n' as a variable?
May we just make it
`description_str = f' """{self.description}""" \n' if self.description else ''`
I think it's more python-like

Reply:
sure I can change that if you prefer - more so just following the way it is already defined in other GQL class modules such as `backend/dataall/base/api/gql/graphql_enum.py` where `n='\n'` is defined as such

since we are only calling on `\n` once here I will just make the change you described above